test/unit/server/middleware/StaticAssetHandler.test.ts

```diff
@@ -213,4 +214,21 @@ describe('A StaticAssetHandler', (): void => {
     const response = createResponse({ eventEmitter: EventEmitter });
     await expect(handler.canHandle({ request, response } as any)).rejects.toThrow();
   });
+
+  it('caches responses when the expires option is set.', async(): Promise<void> => {
+    MockDate.set(0);
```

Comment:
It's quite simply to do without external library so I would remove that one: https://github.com/solid/community-server/blob/7983170795339eae7135bf3011b5727fc0600d56/test/unit/identity/ownership/TokenOwnershipValidator.test.ts#L35-L36
but with a fixed value for now in this case.